tools/ci/run-examples.sh

```diff
@@ -30,6 +30,10 @@ python -m pip install ./wheels/*
 echo "Running local processor quickstart."
 python python/feathub/examples/nyc_taxi.py
 
+wheel_files=`ls ./wheels/*`
```

Comment:
> Would it be more readable to move those three lines right below python -m pip install ./wheels/*?

I think the current approach is better, since it verifies that users need not install feathub[flink] if they just want to use local processor.

> Is the purpose of these lines to install FeatHub from source? If so, can we re-use the approach provided in the README?

Does "the approach provided in the README" refer to the following codes?

```bash
python -m pip install --upgrade feathub-nightly
python -m pip install --upgrade feathub-nightly[flink]
python -m pip install --upgrade feathub-nightly[spark]
```